### ISSUE TITLE
Xcom Armor Human Exclusivity (Discussion)

### ISSUE THREAD

**Issue by Optimism333** [d3athrow/vgstation13#34915]
I'm not planning on taking action towards this as of yet, but I am curious what the general consensus is.

As of current, insectoids have sprites molded for them for the xcom armors: squaddie armor and mysterious armor.

**My question is this:** 
Should xcom armor be human exclusive? I am of the opinion it should be, as an armor design from a human organization created for men and women battling aliens doesn't seem like it should mold to an alien form (certainly at least not bugs and vox).

[discussion]

**Comment by BomberBro** [comment]
Xcom armor makes perfect sense to be human exclusive, there are already grey and vox exclusive armors. I think it gives races a bit more flavor when they have their own unique things, especially for humans which are otherwise seen as generic.

**Comment by kilozombie** [comment]
I like the Original Armor. They make my bug look like a weird fucked up alien prowling around with Science gear. If people really hate the idea I would love to see it moved onto a new item and put in the costume vendor or something, since it now does resemble the old Mysterious Armor sprite more than the humans'. Would like it not totally removed.

**Comment by gurfan** [comment]
Throwing my cents in to say that I agree with Kilo here. The armor is fun to wear if only for the look of it. I would like to still be able to wear it as an alien. I think there should be more human exclusive items but I would prefer if those would be new items instead of things that aliens can already wear.

**Comment by Eneocho** [comment]
Yeah, fits much better as a human exclusive, make a small reward somewhere that has the lone insectoid armor prototype or something so the sprites don't go unused though.

**Comment by Optimism333** [comment]
I'll wait to see if more players have input, but from what I'm reading a compromise everyone could be happy with looks like the following:

- Add the old suits (with the old sprites) that can fit both humans and insectoids to the costume vendor.
- Probably dial them back statwise to the old protection values, or just give them protection values similar to the knight armor (melee = 40, bullet = 5, laser = 5, energy = 5, bomb = 0, bio = 0, rad = 0).
- The new armor suits that come from research will be human exclusive.

This way bugs get to keep their fashion options, and humans get to enjoy exclusivity rights over the new mysterious armor and squaddie armor.
